java/spi.java.hints/src/org/netbeans/modules/java/hints/spiimpl/pm/NFABasedBulkSearch.java

```diff
@@ -448,7 +448,7 @@ public void encode(Tree tree, final EncodingContext ctx, AtomicBoolean cancel) {
                 throw new IllegalStateException(ex);
             }
         }
-        if (cancel.get());
+
```

Comment:
this was likely meant to return if cancel is true. I wouldn't remove this, rather try to fix it in a separate PR.